scilpy/tractanalysis/compute_tract_counts_map.pyx

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
+#cython: profile=False
+
+import cython
+import numpy as np
+cimport numpy as cnp
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def compute_tract_counts_map(array_sequence, tuple shape):
```

Comment:
Don't we have this already in `scilpy/tractanalysis/streamlines_metrics.pyx:def compute_tract_counts_map(streamlines, vol_dims):` ?